job-ocp.yaml

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
What is the difference between  job-ocp.yaml and job.yaml?

Reply:
Few additional volume mounts `"/etc/group"` and `"/etc/passwd"` and service account token mounting
```
serviceAccountName: kube-bench
automountServiceAccountToken: true
```

Reply:
@deebhatia Do we need to include the dependent ServiceAccount and RBAC resources in this YAML file?

Reply:
Hi @mozillazg, thanks for the suggestion. I have included all the resources needed for the job to work fine in same YAML file.